ipa-core/src/report/hybrid.rs

```diff
@@ -673,7 +673,7 @@ where
 /// and `HybridImpressionReport`s so that they can be made indistingushable.
 /// Note: these need to be shuffled (and secret shares need to be rerandomized)
 /// to provide any formal indistinguishability.
-#[derive(Clone, Debug, Eq, PartialEq)]
+#[derive(Clone, Debug, Eq, PartialEq, Default)]
```

Comment:
do you need it? I don't think providing `Default` implementation makes sense anywhere except tests, so `[cfg_attr(test, derive(Default))]` is a better option imo

Reply:
pretty sure it is. this is the compiler error I get without it
```
error[E0277]: the trait bound `IndistinguishableHybridReport<BK, V>: Default` is not satisfied
   --> ipa-core/src/protocol/hybrid/oprf.rs:177:21
    |
177 |       let curve_pts = seq_join(
    |  _____________________^
178 | |         ctx.active_work(),
179 | |         process_slice_by_chunks(input_rows, move |idx, records: ChunkData<_, CONV_CHUNK>| {
180 | |             let record_id = RecordId::from(idx);
...   |
189 | |     .map_ok(Chunk::unpack::<PRF_CHUNK>)
190 | |     .try_flatten_iters()
    | |________________________^ the trait `Default` is not implemented for `IndistinguishableHybridReport<BK, V>`
    ```

Reply:
hmm, something is off here - we shouldn't require to process those reports by chunks

Reply:
nope, we really need it to generate curve points from match keys